### ISSUE TITLE
skr-kyma-to-kyma2-upgrade-aws-dev fails to delete application resources

### ISSUE THREAD

**Issue by pbochynski** [kyma-project/kyma#13110]
**Description**

The job [skr-kyma-to-kyma2-upgrade-aws-dev](https://status.build.kyma-project.io/job-history/gs/kyma-prow-logs/logs/skr-kyma-to-kyma2-upgrade-aws-dev) fails because of:
```
  1) SKR-Upgrade-test
       Test fixtures should be deleted:
     Failed to fetch resource metadata for applicationconnector.kyma-project.io/v1alpha1/Application: connect ECONNREFUSED 127.0.0.1:8080
  Error: connect ECONNREFUSED 127.0.0.1:8080
      at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1159:16)
```

**Comment by tobiscr** [comment]
Seems to be caused by https://github.tools.sap/kyma/backlog/issues/2206

**Comment by khlifi411** [comment]
This will be fixed by merging this pr: #13117

**Comment by pbochynski** [comment]
Update from @janmedrek :
- https://github.com/kyma-project/kyma/blob/1ac23f81391bbed846a72ae389122876c3adbe57/tests/fast-integration/utils/index.js#L947
- https://github.com/kyma-project/kyma/blob/1ac23f81391bbed846a72ae389122876c3adbe57/tests/fast-integration/compass/fixtures.js#L48
- this fails with connection refused error: https://github.com/kyma-project/kyma/blob/1ac23f81391bbed846a72ae389122876c3adbe57/tests/fast-integration/test/fixtures/commerce-mock/index.js#L695

**Comment by janmedrek** [comment]
https://github.com/kyma-project/kyma/pull/13167

**Comment by khlifi411** [comment]
This was resolved by merging #13156
